packages/flutter_test/lib/src/finders.dart

```diff
@@ -639,6 +639,18 @@ class CommonSemanticsFinders {
     );
   }
 
+  /// Finds any [SemanticsNode]s that can scroll in at least one direction.
+  ///
+  /// {@macro flutter_test.finders.CommonSemanticsFinders.viewParameter}
+  SemanticsFinder scrollable({FlutterView? view}) {
```

Comment:
I would probably expect that I can specify weather I want something that scrolls horizontal or vertical (or don't care).

Reply:
Done. Tests added as well.